### DISCUSSION TITLE
Using Bonsai for Timestamp Camera acquisition

### DISCUSSION THREAD

**Discussion by navidt11** [bonsai-rx/bonsai#1018] [Q&A] [answered]
Hi all! I'm Navid and I am a researcher at the University of Kentucky trying to figure out how and if the Bonsai software can help me with my studies. Our lab uses Inscopix for in vivo calcium imaging and Doric Neuroscience Studio for fiber photometry. We were hoping to use Bonsai as a third party software to synchronize data acquisition and camera capture. Is this doable with the programs we have? If anyone has any advice it'd be great. We'd like to get this set up in operant chambers. Cheers!

**Comment by bruno-f-cruz** [comment, marked as answer]
Hi Navid,
Unfortunately both companies do not release code (APIs) to interface with their devices. As a result there aren't any bonsai packages that will allow you to communicate with the cameras you mention, at this point in time.
Cheers,
Bruno

**Reply by navidt11** [reply]
Well that is very unfortunate but I appreciate your response Bruno, thank you!

**Reply by navidt11** [reply]
One last quick question, do you know whether there would be any compatibility with bonsai and Med-PC?

**Reply by bruno-f-cruz** [reply]
I am not familiar with the Med-PC protocol. I did a quick search and it doesn't appear to be open-source, unfortunately. Do you have an example of how to control Med-PC hardware from third-party software? Or even how to interface with other programming languages?
